ovoscope/audio.py

```
@@ -0,0 +1,766 @@
+        self.svc = PlaybackService(
+            bus=self.bus,
+            tts=self.mock_tts,
+            disable_ocp=self.disable_ocp,
+            validate_source=self.validate_source,
+        )
+        # Wire mock_tts to the playback thread created by PlaybackService
+        self.mock_tts.init(self.bus, self.svc.playback_thread)
+
+        # Subscribe lifecycle events for synchronisation
+        self.bus.on("recognizer_loop:audio_output_start",
+                    lambda m: self._audio_output_start.set())
```

Ruff (0.15.5)

[warning] Unused lambda argument: `m`

(ARG005)